app/models/step_by_step_page.rb

```diff
@@ -144,7 +144,7 @@ def links_last_checked_date
   end
 
   def should_show_required_prepublish_actions?
-    has_draft? && !scheduled_for_publishing? && !can_be_published?
+    has_draft? && !scheduled_for_publishing? && !can_be_published? || status.in_review?
```

Comment:
I'd recommend adding some parentheses here to make it very clear which conditions are associated with the AND and which are associated with the OR.